Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/SAMLAssertionBuilderException.java

```diff
@@ -0,0 +1,9 @@
+package gov.hhs.fha.nhinc.callback.openSAML;
+
+public class SAMLAssertionBuilderException extends Exception {
+
+    public SAMLAssertionBuilderException(String message)
+    {
+        super(message);
+    }
+}
```

Comment:
Stylistic suggestion: Files within CONNECT usually end with a final blank line, unless there's a reason not to.